packages/ag-charts-community/src/chart/mapping/themes.test.ts

```diff
@@ -17,7 +16,7 @@ import { themes } from './themes';
 expect.extend({ toMatchImageSnapshot });
 
 /* eslint-disable no-console */
-describe('ThemesValidation', () => {
+describe('validateChartThemeObject', () => {
```

Comment:
Should this be called `'Themes'` to match the other changes matching their file names?

Reply:
Updated